app/helpers/fhir_helper.rb

```diff
@@ -241,6 +241,27 @@ def close_contact_from_fhir(related_person)
     }
   end
 
+  def history_as_fhir(history)
```

Comment:
This isn't capturing `history_type`, I think we can place that in `Provenance.agent.type.coding.code`

Reply:
I actually thought about that, but my concern is figuring out what preset codes would map to our set of history_type values. I'm not sold that Provenance.agent.type.coding.code is a good fit, since the available values seem to correspond more to actors rather than objects, e.g. "A person who verifies the correctness and appropriateness of activity" which definitely isn't gonna fit the same set as "Monitoring Change". Maybe Provenance.activity?

Reply:
Yeah you're totally right, honestly the more I look, it feels like wherever we put it would be a stretch. Maybe we should just take our medicine and make it an extension? What do you think.